style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-image: url(./imges/background\ image.png);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Your background image is not displaying properly in the browser. The reason for this is that you applied the background image style using the `*` selector, which styles every single element on your page. Please remove the style rules on lines 5 - 7 and apply them on the container for your headline section🙏

Your design:

<img width="211" alt="image" src="https://user-images.githubusercontent.com/37116322/227590461-fcfe9ae7-d5e5-48c8-aa6d-767d4af782c8.png">